st2actions/st2actions/handlers/mistral.py

```diff
@@ -31,12 +31,17 @@
 LOG = logging.getLogger(__name__)
 
 
-STATUS_MAP = dict()
-STATUS_MAP[action_constants.LIVEACTION_STATUS_REQUESTED] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_SCHEDULED] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_RUNNING] = 'RUNNING'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_SUCCEEDED] = 'SUCCESS'
-STATUS_MAP[action_constants.LIVEACTION_STATUS_FAILED] = 'ERROR'
+STATUS_MAP = {
```

Comment:
Actually, ignore that.

I see you added some tests for that below - https://github.com/StackStorm/st2/pull/2264/files#diff-6153b0d1287b2dcbcc120c6e037bd783R601